Source/WebKit/Shared/ShareableResource.h

```diff
@@ -50,7 +50,7 @@ class ShareableResourceHandle {
     void encode(IPC::Encoder&) const;
     static WARN_UNUSED_RETURN bool decode(IPC::Decoder&, ShareableResourceHandle&);
 
-    RefPtr<WebCore::SharedBuffer> tryWrapInSharedBuffer() const;
+    RefPtr<WebCore::SharedBuffer> tryWrapInSharedBuffer();
```

Comment:
So also this could be marked as rvalue reference, so that the caller doesn't end up with the contents of `*this` mutated behind the scenes.
So you'd mark this up as being rvalue qualified function:
```
    RefPtr<WebCore::SharedBuffer> tryWrapInSharedBuffer() &&;
```

Then caller would call it like:
```
        m_buffer = WTFMove(m_shareableResourceHandle).tryWrapInSharedBuffer();
``